docs/DeveloperGuide.md

```diff
@@ -323,6 +322,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
     Use case resumes at step 1.
 
+* 3a. The addition of contact causes duplicate contacts.
+
+    * 3a1. System shows an error message. 
+  
+    Use case resumes at step 2.
+
```

Comment:
Should we also have another extension:
`*a At any time, User chooses to cancel adding a contact
      Use case ends.`

Not sure if this is needed, because it is a cli

Reply:
I'll add it